Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,18 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes.git", :tag => "v2.2.1"
+  head "https://github.com/greymd/tmux-xpanes.git", :branch => "master"
+  depends_on "tmux" => :recommended
+
+  def install
+    %w[xpanes tmux-xpanes].each do |cmd|
```

Comment:
Are there any binaries/manpages in `bin` and `man` here that Homebrew _wouldn't_ want to install?

Reply:
`tmux-xpanes` command is actually symbolic link of `xpanes`.
Is it better to avoid to make such the alias?

The reason why `tmux-xpanes` command is also provided is to improve the usability.
Because the users may type `tmux[tab]` on the shell to recall the tmux related commands.

Reply:
Essentially my question is whether there's a reason not to simply do:

```ruby
bin.install Dir["bin/*"]
man1.install Dir["man/*.1"]
```

Reply:
Fixed 5a88a1f
Thank you for your suggestion.